modules/dataControllerModule/index.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * This module validates the configuration and filters data accordingly
+ * @module modules/dataController
+ */
+import {config} from '../../src/config.js';
+import {getHook, module} from '../../src/hook.js';
+import {deepAccess, logInfo} from '../../src/utils.js';
+import {processBidderRequests} from '../../src/adapters/bidderFactory.js';
+
+const LOG_PRE_FIX = 'Data_Controller : ';
+const ALL = '*';
+const MODULE_NAME = 'dataController';
+let dataControllerConfig;
+
+const _logInfo = createLogInfo(LOG_PRE_FIX);
+
+function createLogInfo(prefix) {
+  return function (...strings) {
+    logInfo(prefix + ' ', ...strings);
+  }
+}
+
+function hasValidConfiguration() {
+  dataControllerConfig = config.getConfig(MODULE_NAME);
```

Comment:
may I suggest something like `setupConfig` instead of `hasValidConfiguration`? to make it clear that the function has side effects.